src/main/kotlin/no/nav/familie/ks/sak/api/dto/VilkårsvurderingDto.kt

```diff
@@ -67,7 +67,8 @@ data class VilkårResultatDto(
             utdypendeVilkårsvurderinger = utdypendeVilkårsvurderinger,
             personResultat = vilkårResultat.personResultat,
             vilkårType = vilkårResultat.vilkårType,
-            antallTimer = antallTimer
+            // antallTimer kan ikke være 0
+            antallTimer = if (antallTimer == BigDecimal(0)) null else antallTimer
```

Comment:
Skal ikke være nødvendig med denne sjekken lenger

Reply:
Jeg synes vi kan la det være bare for sikkerhets skyld